### PR TITLE
Load constituency data for EFF, UDM, Good

### PR DESCRIPTION
**Pull request by deanmalan**
Import data with:
```
docker-compose run --rm app ./manage.py south_africa_update_constituency_offices --verbose pombola/south_africa/data/constituencies_and_offices/source_data/2019/eff/eff-2019-data.json --party eff --commit --end-old-offices
docker-compose run --rm app ./manage.py south_africa_update_constituency_offices --verbose pombola/south_africa/data/constituencies_and_offices/source_data/2019/udm/udm-2019-data.json --party udm --commit --end-old-offices
docker-compose run --rm app ./manage.py south_africa_update_constituency_offices --verbose pombola/south_africa/data/constituencies_and_offices/source_data/2019/good/good-2019-data.json --party good --commit --end-old-offices
```

### CODE REVIEW

File: pombola/south_africa/data/constituencies_and_offices/source_data/2019/eff/eff-2019-data.json [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+      "Title": "EFF Constituency Office: Cape Town",
+      "Party": "EFF",
+      "Physical Address": "Office 603, 47 Strand Str, Strand Bld, Cape Town, Western Cape.",
+      "Type": "office",
+      "Source Note": "Economic Freedom Fighters Parliamentary Caucus 2019",
+      "Postal Address": "Office 603, 47 Strand Str, Strand Bld, Cape Town, Western Cape."
+    },
+    {
+      "Province": "Free State",
+      "Source URL": "https://www.pivotaltracker.com/file_attachments/100155939/download?inline=true",
```
Comment (jbothma) [inline pombola/south_africa/data/constituencies_and_offices/source_data/2019/eff/eff-2019-data.json]: will this be the URL used in the popolo data? That'll be a bit weird and we should perhaps find a more "official" place to store this, even if there's a bog + files feature on pa.org.za

File: pombola/south_africa/management/commands/south_africa_update_constituency_offices.py [hunk 2]
```diff
@@ -658,19 +671,45 @@ class Command(LabelCommand):
...
+            end_old_offices = True
+            if not party:
+                print("You need to set a party if you want to end all of a party's old offices")
+                return
 
         global VERBOSE
         VERBOSE = options['verbose']
 
-        contact_source_2013 = "Data from the party (2013)"
+        contact_source_2013 = "Data from the party (2019)"
```
Comment (jbothma) [inline pombola/south_africa/management/commands/south_africa_update_constituency_offices.py]: is this right?

File: pombola/south_africa/data/constituencies_and_offices/source_data/2019/eff/eff-2019-data.json (hunk 1 above)
Comment (deanmalan) [reply pombola/south_africa/data/constituencies_and_offices/source_data/2019/eff/eff-2019-data.json]: Uploaded it to PA and updated the URL :)

File: pombola/south_africa/management/commands/south_africa_update_constituency_offices.py (hunk 2 above)
Comment (deanmalan) [reply pombola/south_africa/management/commands/south_africa_update_constituency_offices.py]: Removed it :)

**Comment by jbothma** [comment]
nice! Ship it!

**Review by jbothma — APPROVED** [review]

**Pull request merged**